test/__snapshots__/liveValidatorTests.ts.snap

```diff
@@ -608,14 +608,14 @@ Object {
         "documentationUrl": "",
         "inner": undefined,
         "jsonPathsInPayload": Array [
-          "$.properties.groups[0]",
+          "$.properties.groups[0].displayName",
```

Comment:
actually looking here it seems we are missing examples for multiple similar paths.

can we add for example at https://github.com/Azure/oav/blob/master/test/liveValidation/payloads/objectAdditionalProperties_input.json#L46 another array entry ( can be a copy paste of the existing one ).
Then in the snapshots for that error we should get 2 paths.